app/registries/foreman/plugin.rb

```diff
@@ -124,7 +124,7 @@ def with_global_js
     def_field :name, :description, :url, :author, :author_url, :version, :path
     attr_reader :id, :logging, :provision_methods, :compute_resources, :to_prepare_callbacks,
                 :facets, :rbac_registry, :dashboard_widgets, :info_providers, :smart_proxy_references,
-                :renderer_variable_loaders
+                :renderer_variable_loaders, :host_ui_description
```

Comment:
I think you need to add `@host_ui_description = []` into constructor and a method for registering that plugins will use. Or did I miss it?

Reply:
It can be left blank. I have added a `describe_host` method that will put a relevant value there